partiql-ast/src/main/java/org/partiql/ast/v1/DataType.java

```diff
@@ -391,6 +396,109 @@ public int code() {
         return code;
     }
 
+    public static DataType valueOf(String value) {
```

Comment:
Maybe annotate both return and parameter as NotNull?

Reply:
Yeah. I will add.